plugin/trino-redis/src/main/java/io/trino/plugin/redis/RedisMetadata.java

```diff
@@ -67,7 +68,8 @@
 
         log.debug("Loading redis table definitions from %s", redisConnectorConfig.getTableDescriptionDir().getAbsolutePath());
 
-        this.redisTableDescriptionSupplier = Suppliers.memoize(redisTableDescriptionSupplier::get)::get;
+        this.redisTableDescriptionSupplier = Suppliers.memoizeWithExpiration(
+                redisTableDescriptionSupplier::get, redisConnectorConfig.getTableDescCacheDuration().toMillis(), MILLISECONDS)::get;
```

Comment:
```suggestion
        this.redisTableDescriptionSupplier = Suppliers.memoizeWithExpiration(
                redisTableDescriptionSupplier::get,
                redisConnectorConfig.getTableDescCacheDuration().toMillis(),
                MILLISECONDS);
```

Reply:
Reminder.

Reply:
Sorry, I ignored the modification of this format, I resubmitted it again.